mappers/blog/post.rb

```diff
@@ -8,15 +8,19 @@ def map(context, entry)
       context.slug = context.title&.downcase.urlize.squeeze('-') # Adds url slugs if they don't exist.
     end
 
-    context.title = context.title&.to_s.squeeze(' ').strip # An author enters ' Great   Title!' and it becomes 'Great Title!'
+    if keys.include?(:title)
+      context.title = context.title&.to_s.squeeze(' ').strip # An author enters ' Great   Title!' and it becomes 'Great Title!'
+    end
 
-    # some sanitizing of blog content
-    context.content = (context.content || '')
-      .gsub('.png)', '.png?w=950)') # restrict image width
-      .gsub('.jpg)', '.jpg?w=950)')
-      .gsub('//images.contentful.com/', '//images.ctfassets.net/') # redirect old assets
-      .gsub('//assets.contentful.com/', '//assets.ctfassets.net/') # see: https://www.contentful.com/blog/2017/12/08/change-of-the-contentful-asset-domain/
-      .gsub('//downloads.contentful.com/', '//downloads.ctfassets.net/')
-      .gsub('//videos.contentful.com/', '//videos.ctfassets.net/')
+    if keys.include?(:content)
```

Comment:
Ah, `(context.content || '')` wasn't working?  I always didn't care—let the marketing team figure out what they broke…but also, this is fine..hopefully they don't break a published blog by deleting the content, but oh well!

Reply:
Yes, `context.content` [broke (See Slack)](https://sharesight.slack.com/archives/C099NCVH9/p1622640708163200)

But when I was hacking, I did the same here in this line... ultimately then the build broke elsewhere. So, I'm not sure if this change now has any advantage.